ur_description/urdf/ur5_robot.urdf.xacro

```diff
@@ -5,6 +5,27 @@
   <!-- ur5 -->
   <xacro:include filename="$(find ur_description)/urdf/ur5.urdf.xacro" />
 
+  <!-- ros2 control argument-->
+  <xacro:arg name="use_ros2_control" default="false"/>
+
+  <xacro:if value="$(arg use_ros2_control)" >
+    <!-- other ros2 control arguments -->
```

Comment:
Let's make this a little more descriptive:

`<!-- Data files required by the UR driver -->`